projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/integration-card/plugin-action-button.tsx

```diff
@@ -50,18 +67,39 @@ const PluginActionButton = ( {
 
 	const tooltipTextActivate = __( 'Activate this plugin', 'jetpack-forms' );
 	const tooltipTextInstall = __( 'Install this plugin', 'jetpack-forms' );
+	const tooltipTextNoInstallPerms = __(
+		'You do not have permission to install plugins.',
+		'jetpack-forms'
+	);
+	const tooltipTextNoActivatePerms = __(
+		'You do not have permission to activate plugins.',
+		'jetpack-forms'
+	);
+
+	const getTooltipText = (): string => {
+		if ( isInstalled && ! canUserActivatePlugins ) {
+			return tooltipTextNoActivatePerms;
+		}
+		if ( ! isInstalled && ! canUserInstallPlugins ) {
+			return tooltipTextNoInstallPerms;
+		}
+		return String( isInstalled ? tooltipTextActivate : tooltipTextInstall );
+	};
 
 	return (
-		<Tooltip text={ isInstalled ? tooltipTextActivate : tooltipTextInstall }>
-			<Button
-				variant="primary"
-				onClick={ handleAction }
-				disabled={ isInstalling }
-				icon={ isInstalling ? <Spinner /> : undefined }
-				__next40pxDefaultSize
-			>
-				{ getButtonText() }
-			</Button>
+		<Tooltip text={ getTooltipText() }>
+			<span style={ { display: 'inline-flex' } }>
+				<Button
+					variant="primary"
+					onClick={ handleAction }
+					disabled={ isDisabled }
+					style={ isDisabled ? { pointerEvents: 'none' } : undefined }
+					icon={ isInstalling ? <Spinner /> : undefined }
+					__next40pxDefaultSize
+				>
+					{ getButtonText() }
+				</Button>
+			</span>
```

Comment:
I had to wrap the Button in a span here because if Tooltip directly wraps Button, and Button is disabled, the tooltip will not show. Tooltips do not show on disabled buttons.